### DISCUSSION TITLE
Add ENS getContentHash function

### DISCUSSION THREAD

**Discussion by FrancoAguzzi** [wevm/viem#1567] [Idea / Feature Request]
Currently there are no methods on the `publicClient` (neither in `publicClient.getENSResolver()`) to support fetching and decoding contentHash. In order to do so, ENS Smart contracts must be called. Considering that ENS is widely spread and that Viem has already shown interest in supporting the DAO through features implementation, as one ENS community fan and Web3 builder I'd like to propose the addition of a `getContentHash()` library function.

**Comment by alextnetto** [comment]
Hey @jxom and @tmm! 😃 Would like to advance this. Thoughts?
